app/admin/card.rb

```diff
@@ -0,0 +1,41 @@
+ActiveAdmin.register Card do
+  index do
+    selectable_column
+    column :id
+    column :user
+    column :original_text
+    column :translated_text
+    column :review_date
+    actions
+  end
+
+  show do
+    attributes_table do
+      row :id
+      row :user
+      row :block
+      row :original_text
+      row :translated_text
+      row :review_date
+      row :image
+    end
+    active_admin_comments
+  end
+
+  form do |f|
```

Comment:
Unused block argument - `f`. You can omit the argument if you don't care about it.